setup.py

```diff
@@ -27,6 +27,7 @@
     from Cython.Build import cythonize
 except ImportError:
     if has_pyx:
+        traceback.print_exc()
```

Comment:
This line is new.

Reply:
This line is my debugging stupid CI failures :(

Reply:
The issue has been fixed, but I'm going to leave this in here since it's more explicit/helpful.